### DISCUSSION TITLE
Stuck in queued

### DISCUSSION THREAD

**Discussion by nicoaudy** [codemagic-ci-cd/codemagic-docs#1480] [Q&A] [answered]
My builds get stuck in the "queued" state. Here's the Build ID: 64111b00bd2e32a362639625

**Comment by nhanlnhth** [comment]
same issue

<img width="1252" alt="Screen Shot 2023-03-15 at 10 23 02" src="https://user-images.githubusercontent.com/12705949/225200075-3f333213-9589-4a19-86ca-48ed81ab6fce.png">

**Reply by large** [reply]
Same here, using mac mini m1 VM (free) machines. Stuck since yesterday...

**Comment by konnyaku256** [comment]
same issue

My Project's Build ID: 64113efd2063280071b50ba3

**Comment by mahdi-ninja** [comment]
Same issue here:
build id: 64114222c01ca61e37c46d1b


I started and cancelled it a few times and changed the build settings but it didn't help. Been like this for the last 4-5 hours

**Comment by dancojocaru2000** [comment]
This seems to be a problem only on the M1 Mac mini machines. On Linux or Mac Pro, builds are working.

**Comment by jaxnz** [comment]
We have the same issue on Mac Mini

**Comment by goodbaes** [comment]
same :()

**Comment by Bikram-Shrestha** [comment]
Same here

**Comment by gowtham-xseed** [comment]
+1

**Comment by evelynloo** [comment]
Same here
build id: 64116d2528d92ef528897509

**Comment by emil-dh** [comment]
+1

**Comment by helinanever** [comment, marked as answer]
Hi everyone, please see the related announcement here: https://github.com/orgs/codemagic-ci-cd/discussions/1482
